lib/src/resources/app_assets.dart

```diff
@@ -15,11 +15,17 @@ final class AppAssets {
   static const genderIcon = 'assets/images/gender.png';
   static const handsUpIcon = 'assets/images/hands_up.png';
   static const moonSliceIcon = 'assets/images/moon_slice.png';
-  static const svgStep1o2Icon = 'assets/svg/step1o2_icon.svg';
+  static const svgStep1o2Icon = 'assets/svgs/step1o2_icon.svg';
   static const createProfileStep1o2Image = 'assets/images/step1o2_icon.png';
   static const imageLogo2 = 'assets/images/logo2.png';
 
   static const bantubeatIcon = 'assets/images/bantubeat_icon.png';
   static const salonpriveIcon = 'assets/images/salon_prive_icon.png';
   static const bzcIcon = 'assets/images/bzc_icon.png';
+
+  static const svgEmojiIcon = 'assets/svgs/emoji.svg';
```

Comment:
```suggestion
  static const svgsEmoji = 'assets/svgs/emoji.svg';
```
Respecte la convention de nommage des fichier stp